requirements-server.txt

```diff
@@ -17,5 +17,5 @@ nose==1.3.7
 # Server-specific
 alembic==1.0.0
 gunicorn==20.0.4
-oauthlib==2.1.0
+oauthlib~=3.1.0
```

Comment:
I think this PR should just be about mypy related changes. Also,  we tried to upgrade `oauthlib` to `v3.1.0` and it broke our authentication. See this recent commit here: https://github.com/codalab/codalab-worksheets/commit/1f2c0ea6fadc0eaf5e86bc3d5b407c146bf348d4

Reply:
yeah, I can roll it back.